README.md

```diff
@@ -2,99 +2,63 @@
+3. (Optional) [BitWarden CLI](https://bitwarden.com/help/cli/)
 
-Clone the repository:
+### Instructions
 
-    git clone git@github.com:commonknowledge/meep-intelligence-hub.git
-    cd local-intelligence-hub
+1. Create your `.env` file. There are two methods: one using BitWarden web, and one using the BitWarden CLI.
 
-Create and edit a .env file using `.env-example` file and then
-update `SECRET_KEY` and `MAPIT_API_KEY`. You can get the latter from https://mapit.mysociety.org/account/signup/
+Using BitWarden web:
+- Download the ".env" attachment from the "Mapped Development .env" item in BitWarden, and place it in this folder.
 
-For the front-end, create and edit a .env file using `nextjs/.env.example` (save it as `nextjs/.env`).
+With the BitWarden CLI:
+- Run the following commands:
 
-Follow the below steps to start up the back-end. After data import, there is an extra step for MEEP
-where the created user needs to be verified in the back-end - see the "Logging In" section.
-
-Start up the front end with: `cd nextjs; npm run dev`.
-
-### Running Docker manually (recommended)
-
-Start the Docker environment:
-
-    docker-compose up
-
-Docker-compose will automatically install dependencies and start the development web server at <https://localhost:8000> when the container is started.
-
-(If Python complains about missing libraries, chances are the Python requirements have changed since your Docker image was last built. You can rebuild it with, eg: `docker-compose build web`.)
-
-### Running Docker via Visual Studio Code (VS Code)
-
-1. Install the Remote-Container extension.
-2. In the VS Code Command pallette run `Remote-Containers: Install devcontainer CLI`
-3. Then in the repo directory, run `devcontainer open`
-
-This will setup the docker containers and provide a Bash prompt into the containers. 
-
-You will need to run these further initial steps:
-
-1. `script/server --development`
-2. When complete, press Ctrl+C to stop the server (this allows you to start the server in debug mode instead).
-
-You should now be able to run the `App` run group.
-
-### Data import
-
-If you’re running Docker manually (recommended) you will need to enter a Bash shell inside the container, in order to run any of the data import commands:
-
-    docker-compose exec web bash
-
-If you’re running Docker via Visual Studio Code, instead, you’ll want to run the commands via the built-in terminal.
-
-You will likely want to create a Django superuser, by running this inside the container:
-
-    script/createsuperuser
+```bash
+bw login
+bw get attachment .env --itemid 064abbb1-e41b-4632-be90-b2270105d4d1
+```
 
-The superuser will be created with the details specified in the `DJANGO_SUPERUSER_*` environment variables. [Read more about how Docker handles environment variables](https://docs.docker.com/compose/envvars-precedence/).
+2. Open the project folder in VSCode. You should receive a notification in the bottom left to open the project in
```

Comment:
Please replace all incidences of "should" with definitives like "will", "is", "are" etc.
They're unnecessary; they leave a linguistic space for the possibility that something may not happen as intended, so priming the user. However, there's no function to this: if the expected thing doesn't happen, then the user knows something is wrong. The "should" doesn't help, it just seeds a background sense of uncertainty.

Reply:
@joaquimds I've just made these changes myself to illustrate what I mean 😁